docs/source/whatsnew/1.3.0.txt

```diff
@@ -0,0 +1,90 @@
+Development
+-----------
+
+:Release: 1.3.0
+:Date: TBD
+
+Highlights
+~~~~~~~~~~
+
+Support for Newer Numpy/Pandas Versions
+```````````````````````````````````````
+
+Zipline has historically been very conservative when updating versions of
+numpy, pandas, and other "PyData" ecosystem packages. This conservatism is
+primarily due to the fact that Zipline is used as the backtesting engine for
+[Quantopian](https://www.quantopian.com/), which means that updating package
```

Comment:
I believe this format is only for markdown. This gets rendered as `.rst`, so the format should be: 
```
`Quantopian <https://www.quantopian.com/>`__
```